### ISSUE TITLE
Indent the subreplies under the posts

### ISSUE THREAD

**Issue by innosflew** [mastodon/mastodon#22798]
### Pitch

Move subreplies within posts further to the right or differentiate them somehow from normal replies. Something like to this:
![](https://i.imgur.com/P8g3hi2.png)

### Motivation

Right now subreplies look exactly the same as normal replies and this makes following who is replying to who harder to follow.

**Comment by timtfj** [comment]
Yes please. Not a huge indent (I'd prefer a smaller one than in your example), but enough to indicate what's happening without pushing deeply nested replies too far to the right in mobile view.

Question: how does one do this without ending up with diagonal conversations? Is limiting the number of levels the only way?

**Comment by innosflew** [comment]
> "Is limiting the number of levels the only way?"

@timtfj The way reddit deals with having too many subreply levels on one page is after you reached a certain amount of levels it hides the rest and if you wan to see the rest then you have to click on "Continue this thread" and it will open the rest of the replies on a new page.
![](https://i.imgur.com/PkcGXQp.png)

**Comment by Truhe2342** [comment]
Here's another example of how to do this without indention with the use of vertical coloured lanes, similar to how git branches are often visualized:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/118095316/222777452-8c235772-1fe3-4ab1-bc04-4c26fdfb1af1.png">
